src/main/java/org/openstreetmap/atlas/checks/validation/linear/lines/GeneralizedCoastlineCheck.java

```diff
@@ -38,8 +38,8 @@
             SHARP_ANGLE_INSTRUCTIONS);
     private static final double MINIMUM_DISTANCE_BETWEEN_NODES = 100;
     private static final double MINIMUM_NODE_PAIR_THRESHOLD_PERCENTAGE = 30.0;
-    private static final double SHARP_ANGLE_THRESHOLD_DEFAULT = 83.0;
-    private static final double DO_NOT_EVALUATE_DEFAULT_CAP = -1;
+    private static final double SHARP_ANGLE_THRESHOLD_DEFAULT = Integer.MAX_VALUE;
+    private static final double DO_NOT_EVALUATE_DEFAULT_CAP = Integer.MAX_VALUE;
```

Comment:
Do you think we need to define 2 static variables with the same value? I can see the descriptive value, but it seems a bit strange still.

Reply:
Yes, it was more for the descriptive value, but I made changes in c7549a8 which hopefully gives us the best of both worlds